mautic/mautic
# [Bug] Plugin content locking (referrer POST) causes 500.

| Q  | A
| --- | ---
| Bug fix? | Yes
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #5789
| BC breaks? | No
| Deprecations? | 

[//]: # ( Required: )
#### Description:
When a redirection should occur by ajax (such as "permission denied" or "content locked" situations) and the user is using a plugin (Focus plugin for example), the user may encounter a 500 error. The `referrrerPostAction` cannot find the class intended to handle the action. A minor change should fix this.

#### Steps to test this PR:
1. Attempt to edit the same Campaign (core) as two different users, no 500 error should occur.
2. Attempt to edit the same Focus (plugin) as two different users, no 500 error should occur.
